src/jdk.management.agent/share/classes/jdk/internal/agent/Agent.java

```diff
@@ -36,7 +36,6 @@
 import java.net.MalformedURLException;
 import java.net.UnknownHostException;
 import java.nio.BufferUnderflowException;
-import java.security.AccessController;
 import java.security.PrivilegedAction;
```

Comment:
It doesn't look like java.security.PrivilegedAction is needed.

Reply:
thanks, and found one more in PlatformMBeanProviderImpl.java, will add it to the PR.